src/codegen/constant_translator.cpp

```diff
@@ -68,6 +68,9 @@ codegen::Value ConstantTranslator::DeriveValue(CodeGen &codegen,
       len = codegen.Const32(str.length());
       break;
     }
+    case type::Type::TypeId::ARRAY: {
+      break;
```

Comment:
It might be better to throw an exception here to indicate the codegen doesn't support arrays.